src/Microsoft.Identity.Web/WebAppExtensions/MicrosoftIdentityWebAppAuthenticationBuilderExtensions.cs

```diff
@@ -521,6 +521,9 @@ public static class MicrosoftIdentityWebAppAuthenticationBuilderExtensions
             options.ForwardSignIn = mergedOptions.ForwardSignIn;
             options.ForwardSignOut = mergedOptions.ForwardSignOut;
             options.ForwardDefaultSelector = mergedOptions.ForwardDefaultSelector;
+#if NET8_0_OR_GREATER
+            options.TimeProvider = mergedOptions.TimeProvider;
```

Comment:
Do we also need to update the MergedOptions class?

Reply:
done